client/src/components/recommended-action-button.tsx

```diff
@@ -7,8 +7,8 @@ The full terms of this copyright and license should always be found in the root
 import { Button, Typography } from "@material-ui/core";
 import React from "react";
 import { navigate } from "gatsby";
-import { useWithReviewAnswerState } from "hooks/graphql/use-with-review-answer-state";
-import { Answer, MentorType, Status, UtteranceName } from "types";
+import { Answer, Mentor, MentorType, Status, UtteranceName } from "types";
```

Comment:
We need to get out of habit of throwing tons of model logic into our components (presentation)

Can this be a folder `./components/recommended-action/`

...and then two files within:

 - recommended-action-button.tsx (just the presentation)
 - use-recommended-action.ts (should not need the tsx extension, all the data and logic are here)

Reply:
ended up putting this and mentor card in a folder, split off sub-components and logic functions